lib/tasks/ingestor.rake

```diff
@@ -41,9 +43,13 @@ namespace :fromthepage do
       document_upload.save
     end
 
-    if SMTP_ENABLED    
+    if SMTP_ENABLED
+      begin
         SystemMailer.upload_succeeded(document_upload).deliver!
         UserMailer.upload_finished(document_upload).deliver!
+      rescue *SMTP_ERRORS => e
+        print "SMTP Failed: Exception: #{e.message}"
```

Comment:
The *SMTP_ERRORS list comes from ErrorHelper, although it isn't explicitly called.  I didn't call log_smtp_error from within ErrorHandler because the rake task doesn't recognize the logger call from that method.  I also thought it made sense to have any errors go into the process log, since that's where the rest of the logging goes.